build.gradle

```diff
@@ -82,7 +85,7 @@ subprojects {
 
     uploadArchives {
         onlyIf {
-            name in ['exporter-trace']
+            name in ['exporter-trace', 'exporter-metrics', 'opencensus-migration']
```

Comment:
Please ignore, I need to pull in the latest master, will delete once that's done